docs/imagecustomizer/README.md

```diff
@@ -31,99 +33,10 @@ Disadvantages:
   So, some customizations aren't possible using the Image Customizer.
```

Comment:
nit - doesn't seem like a good way to start the docs with what can't be done. 
Maybe instead call out what is supported and refer to the docs. and maybe rephrase this to state the fact that `customizations are done offline without booting the image ... `

Reply:
I do want to be somewhat frank with what can and can't be done, as this might help reduce customer surprises/pain.

That being said, I did rephrase this to be less negative while still conveying the same information.